daemon/graphdriver/copy/copy.go

```diff
@@ -15,26 +15,75 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-type copyFlags int
+// FICLONE ioctl number -- defined in <linux/fs.h> -- We don't include it
+// because it'd break builds with older versions of kernel headers
+const ficlone = 0x40049409
```

Comment:
Isn't this arch-specific?

Reply:
It is (ish). It applies to all of the architectures other than:
* Power
* Alpha
* MIPS
* Parisc
* Sparc

I can fix this, if you think these architectures should be supported.

Reply:
At least ppc64le needs to be supported.
For other architectures, please disable the added code?

Reply:
Okay, I can actually fix it by adding the IOW in ifndef.